### ISSUE TITLE
no-unexpected-multiline: Error is triggered on multi-line conditional statement or multi-line property access

### ISSUE THREAD

**Issue by nfriend** [buzinas/tslint-eslint-rules#63]
The following triggers a linting error when the `no-unexpected-multiline` rule is `true`:

```typescript
var a, b, c;
if (a === 4
    && (b === 3 || c === 8)) {
}
```

Similarly, a multi-line property access triggers this error:

```typescript
var a;
a
.something();
```

These examples don't trigger linting errors when tested on the [eslint.org demo page](http://eslint.org/demo/).
